engines/director/detection_tables.h

```diff
@@ -672,6 +677,7 @@ static const PlainGameDescriptor directorGames[] = {
 	{ "aapelin1",			"Aapelin ABC: Hurjistunut Pölynimuri" },
 	{ "kcd",				"Kompuutteri Kaikille" },
 	{ "kultapossu",			"Kultapossu ja Leo Leijona: Sateenkaarivarkaat" },
+	{ "muumitpiilosilla",	"Muumit Piilosilla" },
```

Comment:
My first thought was that this should be under Swedish as "Kurragömma med Mumintrollen". Though the author, Tove Jansson, was indeed Finnish, the books were originally published in Swedish. The games were distributed in all Nordic countries, originally developed by Bullhead AB of Stockholm and published by WSOY Multimedia, later Nordic Softsales, also based in Sweden.

However, the series is indeed beloved in Finland. Perhaps, if you haven't already done this, we should consult with someone from that part of the world?

Can you also use the game id `mumin1`, as this is the first in the series (or `muumit1` if we decide to leave it as Finnish)?

Reply:
Hah, I feel like that's getting into some real politics there. The other places I've seen reference this use the Finnish title, but you're right if it's a Swedish developer/publisher maybe the Swedish title should be the primary.

Reply:
Let's make it Swedish since that follows our conventions. If someone objects, or if we can naturally bring it up in conversation with a Swede or Finn without getting into a heated row, then maybe we'll adjust it later on. 🙂

Reply:
Done.